float.js

```diff
@@ -190,6 +190,17 @@ const showFloat = function(listingId) {
         let seedDiv = floatDiv.querySelector('.itemseed');
         if (seedDiv) seedDiv.innerText = `Paint Seed: ${itemInfo.paintseed}`;
 
+        // Set the wear value for each sticker
+        for (const stickerIndex in itemInfo.stickers) {
```

Comment:
You should follow Google's style guide for javascript. Use a `for-of` or keep an index variable and just increment it.

https://google.github.io/styleguide/jsguide.html#features-for-loops